File: eguzman9.java
//Eduardo Guzman


import java.util.Random;
import java.util.Scanner;

public class Main
{
	public static void main(String[] args)
	{
		Random random = new Random();
		int answer = 1 + random.nextInt(20);
		int guess = 0;
		Scanner scanner = new Scanner(System.in);

		while (guess < 1 || guess > 20)
		{
			System.out.println("Guess a random number between 1 - 20:");
			guess = scanner.nextInt();
			if (guess < 1 || guess > 20)
			{
				System.out.println("NOT IT CHEIF! Try another number between 1 - 20!");
			}
		}
		
		if (guess == answer)
		{
			for (int i = 0; i < guess; i++)
			{
			System.out.println("NOICE!");
		}
	}
	
	else
	{
		System.out.println("Better luck next time pal... The number was : " + answer);
		if (guess < answer)
		{
			System.out.println("Too low");
		}
		else
		{
			System.out.println("Too high");
		}
	}

	scanner.close();
	}
}

File: eguzman9.c
// Eduardo Guzman


#include<stdlib.h>
#include<stdio.h>
#include<time.h>

int main(void)
{

	srand(time(NULL));

	int answer = 1 + rand() % 20;
	int guess = 0;

	while (guess < 1 || guess > 20)
	{
		printf("Guess a number between 1 and 20:\n ");
		scanf("%d" , &guess);

		if (guess < 1 || guess > 20)
		{
			printf("Invalid! Please try a number between 1 and 20! \n");
		}
	}

	if (guess == answer)
	{
		for (int i = 0; i < guess; i++)
		{
			printf("NOICE!\n");
		}
	}
	else
	{
		printf("Better luck next time bud... The number was %d. \n", answer);

		if (guess < answer)
		{
			printf("Too low\n");
		}
		else
		{
			printf("Too high\n");
		}
	}


	return EXIT_SUCCESS;
}
